appstoreserverlibrary/models/ExtendRenewalDateRequest.py

```diff
@@ -17,9 +17,9 @@ class ExtendRenewalDateRequest:
     extendByDays: Optional[int] = attr.ib(default=None)
     """
     The number of days to extend the subscription renewal date.
-    
+    The number of days is a number from 1 to 90.
```

Comment:
This doesn't follow the style of the rest of the library, where only the first sentence of a field's documentation is listed. Could you please remove this bit. Discussion about the style of documentation would probably be best in a separate issue

Reply:
done